docs/.vuepress/config.js

```diff
@@ -1,6 +1,6 @@
 // .vuepress/config.js
 
-const DEPLOY_DOMAIN = 'https://docs.ipfs.io'
+const DEPLOY_DOMAIN = 'https://docs.ipfs.tech'
```

Comment:
:information_source: this alone takes care of setting correct [canonical link](https://en.wikipedia.org/wiki/Canonical_link_element) in page headers